pkg/splunk/enterprise/names.go

```diff
@@ -106,11 +97,11 @@ const (
 	protoHTTPS = "https"
 	protoTCP   = "tcp"
 
-	// Volume name for shared volume between init and splunk containers
-	appVolumeMntName = "init-apps"
+	// Volume name for splunk containers to store apps temporarily
+	appVolumeMntName = "apps-staging-volume"
 
-	// Mount location for the shared app package volume
-	appBktMnt = "/init-apps/"
+	// Mount location on splunk pod for the app package volume
+	appBktMnt = "/apps-staging-volume/"
```

Comment:
Can we mount it as something like `/operator-staging/`, then create a directory, something like `app-framework`, so that the same mount location can be used for any future operator usage?

Reply:
We can do that but I am averse to overloading the same mount location for many future things, wanted to keep it clean. But lets see what others have to say about this :)